vexim/login.php

```diff
@@ -59,39 +60,43 @@
   print $cryptedpass . "<br>\n";
   */
 
-	# if they have the wrong password bail out
-	if ($cryptedpass !== $row['crypt']) {
-		header ('Location: index.php?login=failed');
-		die();
-	}
+  # if they have the wrong password bail out
+  if ($cryptedpass !== $row['crypt']) {
+    header ('Location: index.php?login=failed');
+    die();
+  }
 
-	# populate session variables from what was retrieved from the database (NOT what they posted)
-    $_SESSION['localpart'] = $row['localpart'];
-    $_SESSION['domain'] = $row['domain'];
-    $_SESSION['domain_id'] = $row['domain_id'];
-	$_SESSION['crypt'] = $row['crypt'];
-    $_SESSION['user_id'] = $row['user_id'];
+  # populate session variables from what was retrieved from the database (NOT what they posted)
+  $_SESSION['localpart'] = $row['localpart'];
+  $_SESSION['domain'] = $row['domain'];
+  $_SESSION['domain_id'] = $row['domain_id'];
+  $_SESSION['crypt'] = $row['crypt'];
+  $_SESSION['user_id'] = $row['user_id'];
 
-	# redirect the user to the correct starting page
-	if (($row['admin'] == '1') && ($row['type'] == 'site')) {
-                if($_POST['crypt']=="CHANGE") {
-                  header ('Location: sitepassword.php');
-                  die();
-                }
-		header ('Location: site.php');
-		die();
-	} 
-	if ($row['admin'] == '1') {
-		header ('Location: admin.php');
-		die();
+  # redirect the user to the correct starting page
+  if (($row['admin'] === '1') && ($row['type'] === 'site')) {
+    if($_POST['crypt'] === "CHANGE") {
+      header ('Location: sitepassword.php');
+      die();
     }
-	if (($row['domainenabled'] == '0')) {
-		header ('Location: index.php?domaindisabled');
-		die();
-}
-	
-	# must be a user, send them to edit their own details
-	header ('Location: userchange.php');
-	
+    header ('Location: site.php');
+    die();
+  }
+  if ($row['admin'] === '1') {
+    header ('Location: admin.php');
+    die();
+ }
+ if (($row['userenabled'] === '0')) {
+   header ('Location: index.php');
```

Comment:
I think there is a snippet in the index.php which already adds this 
information (account disabled).

On 2015-11-13 16:53, Rimas Kudelis wrote:
> In vexim/login.php [1]:
> 
>> -        header ('Location: index.php?domaindisabled');
>> -        die();
>> -}
>> -
>> -    # must be a user, send them to edit their own details
>> -    header ('Location: userchange.php');
>> -
>> +    header ('Location: site.php');
>> +    die();
>> +  }
>> +  if ($row['admin'] === '1') {
>> +    header ('Location: admin.php');
>> +    die();
>> + }
>> + if (($row['userenabled'] === '0')) {
>> +   header ('Location: index.php');
> 
> Perhaps redirect it with a param similarly to ?domaindisabled?
> 
> --
> Reply to this email directly or view it on GitHub [2].
> 
> 	*
> 
> Links:
> ------
> [1] https://github.com/vexim/vexim2/pull/95#discussion_r44797193
> [2] https://github.com/vexim/vexim2/pull/95/files#r44797193